### ISSUE TITLE
Popup not blocked on https://www.google.com/amp/s/amp.reddit.com/*

### ISSUE THREAD

**Issue by cbwee** [easylist/easylist#7529]
From https://github.com/uBlockOrigin/uAssets/issues/8786.

### List the website(s) you're having issues:
`https://www.google.com/amp/s/amp.reddit.com/r/explainlikeimfive/comments/ecrzvp/eli5_what_are_amp_pages_and_whats_bad_about_them/`
(Open it in a mobile browser, or a desktop browser with mobile user agent)

### What happens?
1. Open Brave Browser on Android, enable Fanboy Annoyances List and uBlock Annoyances List in `Brave://adblock` .
2. Open this google search result in Brave Android, `https://www.google.com/search?q=amp+reddit` .
3. Click the second search result, which has an AMP symbol, and is an AMP page.
4. This page will be opened.
`https://www.google.com/amp/s/amp.reddit.com/r/explainlikeimfive/comments/ecrzvp/eli5_what_are_amp_pages_and_whats_bad_about_them/`
5. The popup is not blocked, as you can see in the screenshot.

6. Now test with Iceraven Browser, with uBlock Origin installed (also install Google Search Fixer).
7. In Iceraven Browser, enable uBlock filters - Annoyances and Fanboy's Annoyance.
8. Repeat steps 2 to 5, the popup is blocked, but the background of it is not blocked, as you can see in the second screenshot.

### Screenshot(s)
1. Brave Browser on Android
<img src="https://user-images.githubusercontent.com/34395942/112597639-57fe3880-8e48-11eb-9032-fdeab8b5107b.png" height="500">
2. Iceraven Browser
<img src="https://user-images.githubusercontent.com/34395942/112597989-e5da2380-8e48-11eb-8ec6-5825963da360.jpg" height="500">

### List Subscriptions you're using:
I have enabled Fanboy's Annoyance and uBlock filters - Annoyances, in both browsers.

### Your settings
I have tested it when uBlock Origin and Google Search Fixer are the only extensions in Iceraven, I also tested it in Brave Browser when it has the default settings, with Fanboy Annoyances List and uBlock Annoyances List enabled.

- OS/version: Android 10 with MIUI 12.
- Browser/version: Brave 1.22.69 Android / Iceraven 1.7.1
- Adblock Extension/version: Brave Ad Block / uBlock Origin v1.34.0

### Other details:
The popup still appears after 
https://github.com/uBlockOrigin/uAssets/pull/8690/commits/82fac4c60b3ff45d760089597fb9864ce1d4f784
and
https://github.com/easylist/easylist/commit/63d4fb28c1878048f55f62298aade1f7d5eef3f6

**Comment by krystian3w** [comment]
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

 So gone from `Fanboy Annoyance`.

IMO page block scroll so why closed in uAssets?

**Comment by krystian3w** [comment]
for uBlock Origin can use:

`amp-reddit-com.cdn.ampproject.org#?#.AppSelectorModal__body:upward(1)`

**Comment by ryanbr** [comment]
Fixed in Annoyances
